plugins/reporters/spdx/src/main/kotlin/SpdxDocumentReporter.kt

```diff
@@ -167,3 +166,12 @@ private fun SpdxExpression.getLicenseRefExceptions(result: MutableSet<String>) {
         else -> {}
     }
 }
+
+private fun String.patchSpdx23To22() =
+    replace(spdxVersion22Regex, "$1SPDX-2.2$2")
+        .replace("PACKAGE-MANAGER", "PACKAGE_MANAGER")
+        .replace("PERSISTENT-ID", "PERSISTENT_ID")
```

Comment:
Would it make sense to use the same approach as for the `spdxVersion` version, e.g. include the property name in the regex?